### ISSUE TITLE
Poor performance on BungeeNeRF

### ISSUE THREAD

**Issue by shawnFuu** [city-super/Scaffold-GS#106]
Hi. I have reproduced your code on BungeeNeRF dataset.I totally used the same experiment settings. However, the performance is poorer than those reported in the paper. I show these below:

![Image](https://github.com/user-attachments/assets/e52ec4f0-7eab-471f-be76-267dea8b654c)

That's weird. I really appreciate your work, which I have followed it for a long time. Do you have any suggestions? If any replies can be given by anyone, it would be very heplful for me!

**Comment by cskrren** [comment]
Could you please tell me what the training results are like under the normal settings (taking 1 test out of 8)?

**Comment by shawnFuu** [comment]
I followed the settings of BungeeNeRF and yours. I have reproduced the right results when i ablated the modules in my own work with only baseline left(Scaffold-GS), though the results were still weird when i ran the code you provided.

**Comment by shawnFuu** [comment]
> Could you please tell me what the training results are like under the normal settings (taking 1 test out of 8)?

I understand now. The parameter "lod" cannot be ignored. If it is positive, the dataset will be divided into training set\test set according to "lod". So the performance will be worse.
